.github/actions/run-tests-in-container/Dockerfile

```
@@ -9,12 +9,20 @@
 # Base Image
 FROM alpine
 
+
 # Testing Dependencies
+# if wanting to install cpp-coveralls via pip install --user cpp-coveralls, these deps are required:
+# python3-dev py3-pip libffi-dev rust cargo openssl-dev git 
+
+# install gcovr from the package manager, which gets most required dependencies (avoiding the need for libxml2-dev, libxslt-dev and setting the include path.)
+# then install using pip to get the 5.x line with support for coveralls formatted reports.
 RUN ["ash", "-c", "\
     apk add --no-cache \
       gcovr \
+      py3-pip python3-dev \
```

Can these each occupy their own line?
```suggestion
      py3-pip \
      python3-dev \
```

---

ok, I thought the logical grouping made sense